.github/workflows/qa.yml

```diff
@@ -0,0 +1,38 @@
+name: Node CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```

Comment:
I'd better use fixed version/release if possible. (same idea as for JS packages)